api/build.go

```diff
@@ -553,11 +556,45 @@ func GetOrgBuilds(c *gin.Context) {
 	// nolint: gomnd // ignore magic number
 	perPage = util.MaxInt(1, util.MinInt(100, perPage))
 
+	// See if the user is an org admin to bypass individual permission checks
+	perm, err := source.FromContext(c).OrgAccess(u, o)
+	if err != nil {
+		logrus.Errorf("unable to get user %s access level for org %s", u.GetName(), o)
+	}
+	var excludeList []int64
+	if perm != "admin" {
+		// Get a list of private repos to filter out if the user does not have permission to them
+		privateRepos, err := database.FromContext(c).GetOrgPrivateRepoList(o)
```

Comment:
I think we're missing a couple parameters to provide to this function specifically around pagination.

```suggestion
		privateRepos, err := database.FromContext(c).GetOrgPrivateRepoList(o, page, perPage)
```

Reply:
@jbrockopp @wass3r 

After thinking about this for a while, maybe we just want to exclude builds and repos that are private from the Org pages. 
We took this approach to avoid data leakage, however I think the overhead of checking permissions by hitting the github api will not scale well.

Reply:
@JayCeeJr I'm not opposed to that approach and I think that's a fair way to handle it 👍 

We can always explore attempting to add this functionality in at a later date if it's found valuable by our users.

And perhaps in the future we'll be storing user permissions for orgs and repos in our database?

Thinking that maybe it would help resolve some concerns with taxing the GitHub API.

Reply:
@jbrockopp 
I took a middle-ground approach to this. I check if the user is an admin, if not i only show public repos for the org. Same logic applies to the builds.

Reply:
May need to utilize the [list organization repos](https://docs.github.com/en/rest/reference/repos#list-organization-repositories) endpoint or [graphql](https://github.com/shurcooL/githubv4) in the future.